Added capabilty to read ORDER and ignore whitespace after null specifier. (#11593)
astropy/astropy: docs/changes/io.ascii/11593.bugfix.rst

> Made sure that the  CDS reader does not ignore order specifier that 

Reviewer: ```suggestion
Made sure that the CDS reader does not ignore an order specifier that
```

> maybe present after the null specifier '?'. Also made sure it only 

Reviewer: ```suggestion
maybe present after the null specifier '?'. Also made sure that it
```

> checks null values when a '=' symbol is present, and read description 

Reviewer: ```suggestion
checks null values only when an '=' symbol is present, and reads description
```
